api/v1/user_manager_service.proto

```diff
@@ -66,6 +94,30 @@ service UsersService {
       delete: "/v1/users/api_keys/{id}"
     };
   }
+
+  rpc CreateOrganization(CreateOrganizationRequest) returns (Organization) {
+    option (google.api.http) = {
+      post: "/v1/organizations"
+      body: "*"
+    };
+  }
+  rpc ListOrganization(ListOrganizationRequest) returns (ListOrganizationResponse) {
+    option (google.api.http) = {
+      get: "/v1/organizations"
+    };
+  }
+  rpc DeleteOrganization(DeleteOrganizationRequest) returns (DeleteOrganizationResponse) {
+    option (google.api.http) = {
+      delete: "/v1/organizations/{id}"
+    };
+  }
+
+  rpc AddUserToOrganization(AddUserToOrganizationRequest) returns (AddUserToOrganizationResponse) {
+    option (google.api.http) = {
+      post: "/v1/organizations/{organization_id}/users"
```

Comment:
Is this following OpenAI API spec? If not, my preference is to have a custom method URL for this as "Add" is not strictly "Create".

```
      post: "/v1/organizations/{organization_id}:addUser"
```

https://cloud.google.com/apis/design/custom_methods

Or another option would be to rename the function to `CreateUserOrganization` (same name as the `store` function).

Reply:
OpenAI only provides invite API, so this method does not follow them. I'll fix it using the custom method.